arches/app/templates/views/components/widgets/reference-select.htm

```diff
@@ -0,0 +1,55 @@
+{% extends "views/components/widgets/base.htm" %}
+{% load i18n %}
+
+{% block form %}
+<div class="row widget-wrapper" data-bind="class: nodeCssClasses">
+    <div class="form-group">
+        <span class="control-label widget-input-label" data-bind="text:label, class: nodeCssClasses"></span>
+        <!-- ko if: node -->
+        <i data-bind="css: {'ion-asterisk widget-label-required': node.isrequired}"></i>
```

Comment:
I know they're all like this, but let's use this opportunity to get an aria-label on the icon.